query-engine/connector-test-kit-rs/query-engine-tests/tests/new/ref_actions/on_delete/set_default.rs

```diff
@@ -2,7 +2,7 @@
 use indoc::indoc;
 use query_engine_tests::*;
 
-#[test_suite(suite = "setdefault_onD_1to1_req", exclude(MongoDb, MySQL))]
+#[test_suite(suite = "setdefault_onD_1to1_req", exclude(MongoDb, MySQL, TiDB))]
```

Comment:
Why does TiDB need to be excluded here, but not Vitess?

Reply:
I have no idea about that why Vitess can pass this test.

It should throw error as following:

```
thread 'new::ref_actions::on_delete::set_default::one2one_req::delete_parent' panicked at 'called `Result::unwrap()` on an `Err` value: ConnectorErrorImpl { user_facing_error: Some(KnownError { message: "error: Error validating: Invalid referential action: `SetDefault`. Allowed values: (`Cascade`, `Restrict`, `NoAction`, `SetNull`)\n  -->  schema.prisma:22\n   | \n21 |     parent_id Int    @default(2) @unique\n22 |     parent    Parent @relation(fields: [parent_id], references: [id], onDelete: SetDefault)\n   | \n", meta: Object {"full_error": String("error: Error validating: Invalid referential action: `SetDefault`. Allowed values: (`Cascade`, `Restrict`, `NoAction`, `SetNull`)\n  -->  schema.prisma:22\n   | \n21 |     parent_id Int    @default(2) @unique\n22 |     parent    Parent @relation(fields: [parent_id], references: [id], onDelete: SetDefault)\n   | \n")}, error_code: "P1012" }), message: Some("error: Error validating: Invalid referential action: `SetDefault`. Allowed values: (`Cascade`, `Restrict`, `NoAction`, `SetNull`)\n  -->  schema.prisma:22\n   | \n21 |     parent_id Int    @default(2) @unique\n22 |     parent    Parent @relation(fields: [parent_id], references: [id], onDelete: SetDefault)\n   | \n"), source: None, context: SpanTrace [] }
error: Error validating: Invalid referential action: `SetDefault`. Allowed values: (`Cascade`, `Restrict`, `NoAction`, `SetNull`)
  -->  schema.prisma:22
   | 
21 |     parent_id Int    @default(2) @unique
22 |     parent    Parent @relation(fields: [parent_id], references: [id], onDelete: SetDefault)
   | 
```

Reply:
That makes a lot of sense. We should double check what is going on here.

Reply:
@Mini256  or @janpio  what are the next steps required here ? I know some investigation is needed but who plans on doing it ?

Reply:
Vitess is probably somehow completely excluded here, as it does not support Foreign keys at all (in the configuration we test and use). So possibly Vitess does not even run these tests somehow? We'll need to look at our test setup to understand that.

Reply:
@Mini256  and @janpio  is that not the same case with TiDB as well, we don't support Foreign Keys as well. Should we also not exclude these tests ?

Reply:
There is no exclusion for Vitess above, only MongoDB and MySQL. That is exactly what confuses me.

Reply:
Resolved it, because TIDB does not support SET DEFAULT like MySQL